msteams-platform/concepts/tabs/tabs-overview.md

```diff
@@ -1,27 +1,33 @@
 ---
-title: Get started with tabs
-description: Describes how to get started with tabs in Microsoft Teams
-keywords: get started tabs teams
+title: Develop tabs
+description: Describes how to get started developing tabs in Microsoft Teams
+keywords: teams tabs development
 ---
 
-# Get started with tabs for Microsoft Teams
+# Develop tabs for Microsoft Teams
 
 Tabs in Microsoft Teams allow you to display rich interactive web content. You can build a Microsoft Teams tab from scratch or adapt your existing web-app experience.
 
+![Example of a tab showing data, alongside a conversation about the tab data](~/assets/images/tab_example.png)
+
 Microsoft Teams supports tabs in two scopes:
 
 * **Team scope**&emsp;Tabs in channels allow teams to interact with your shared experience. Currently, all tabs in channels are *configurable tabs*&mdash;a user configures the content of your tab experience when the tab is first added to a channel.
-* **Personal scope**&emsp;Personal tabs allow users to interact with your experience privately. Currently, all personal tabs are *static tabs*&mdash;content that is relevant to individual users.
+* **Personal scope**&emsp;Tabs in the personal scope allow users to interact with your experience privately. Currently, all such tabs are *static tabs*&mdash;content that is relevant to individual users.
+
+## What you need to know: Configurable tabs
 
-## Build a configurable tab
+A *configurable tab* becomes part of a channel and provides a single kind of information to a team. For example, the Planner tab for a channel contains a single plan; the Power BI tab maps to a specific report. Users can drill down to the relevant context, but they should not be able to navigate outside the tab. The Power BI tab, for instance, doesn't enable navigation to other Power BI reports&mdash;but it does enable the **Go to website** button that launches the report in the main Power BI website.
 
 *  [Create the configuration page](~/concepts/tabs/tabs-configuration): For configurable tabs, you must provide a configuration page to present options and gather information so users can customize the content and experience with your tab. This iframe'd HTML page is displayed when a user first adds the tab to a channel.
```

Comment:
iframe'd? We thought of changing that?

Reply:
Yep. I even had an action item 'nd all that—but I missed this one. Fixed.